apps/synapse-portal-framework/src/components/PortalSearch/PortalSearchTabs.tsx

```diff
@@ -0,0 +1,102 @@
+import { Box, Tab, Tabs, useMediaQuery, useTheme, Chip } from '@mui/material'
+
+import { useNavigate, useLocation } from 'react-router-dom'
+import { CustomScrollButton } from '../Explore/ExploreWrapperTabs'
+export type PortalSearchTabConfig = {
+  path: string
+  title: string
+  count?: number
+}
+
+export type PortalSearchTabUIProps = {
+  tabConfig: PortalSearchTabConfig[]
+}
+export function PortalSearchTabs(props: PortalSearchTabUIProps) {
```

Comment:
Glad the Explore tab UI translated so easily. I feel like much more could be done here to reduce duplicate code/styles with ExploreWrapperTabs.tsx-maybe create some `styled` or other reusable sub-components?

Reply:
That's a great idea